File: python/BaselineFitting.py
import scipy.interpolate as sci
import numpy as np
from matplotlib.pyplot import plot

def BaselineFitting(resource, PMIN, TMIN):
    new_x = np.arange(0, resource.size, 1);
    new_y = sci.spline(TMIN,PMIN,new_x);
    re = resource-new_y;
    return re;

File: matlab_code/BaselineFitting.m
function re = BaselineFitting(resource, PMIN, TMIN)
    %三次样条插值-再次处理基线漂移
    xx = 1:1:length(resource);
    baseline=interp1(TMIN,PMIN,xx,'spline');
    re = resource-baseline;
end